legacy-backend-rust/data/accounts.json

```diff
@@ -0,0 +1,279 @@
+[
```

Comment:
I wonder if there's any value in sharing these to make sure both backends send the exact same data.

Reply:
Yeah I wanted to share them, in a parent directory, but couldn't be bothered to then tar them into each Docker build (as they'd be outside the context). We can evolve this later I think.